### ISSUE TITLE
D&D v4.0.x/2024 Known Issues

### ISSUE THREAD

**Issue by MrPrimate** [MrPrimate/ddb-importer#505]
This is a list of Known Issues for DDB Importer in the 4.0.x System (DDB Importer v6.0.0 and higher):

- [x] 2024 Features and spells are not parsed correctly 
- [x] Automation effects will be applied regardless of settings, if the modules are enabled (ATL, midi, dae, active auras etc)
- [x] Feat Muncher is not enabled
- [x] Species Muncher is not enabled
- [x] Class Muncher is not enabled
- [x] Background Muncher is not enabled
- [ ] Not all Spells have the same out of the box effects as in v5 (Please see list below)
- [x] Not all Items have the same out of the box effects as in v5 (Please see list below)
- [x] Not all Features have the same out of the box effects as in v5 (Please see list below)
- [x] 2024 Rangers do not get their full spells list (they are now prepared casters)
- [x] 2014 spells/items get imported into the same compendium folders as the 2024, provide a way to import into a Legacy folder
- [x] There is an icon missing for.... (please add suggestions below)
- [x] Some effects won't generate duration correctly
- [x] New Summon Spells are not configured correctly
 - [ ] Updates to back to DDB are very hit and miss.
 - [x] Resource retention for items does not always work, depending on activity.
 - [ ] Spell Sniper range increase is not applied to imported characters for either 2014 or 2024
 - [ ] Some Monster Features need parsing corrections, please note any below that are not already noted.
 - [x] There is no option (yet) to determine which version of the spell gets applied to monsters.
 - [x] Spells in the DDB Spells compendium are not added to the Spell Lists.
 - [ ] You can't apply DDB Importer effects to existing documents.
 - [x] You can't apply CPR effects to documents (this enhancement is coming).

Please feel free to comment with any specifics or instances where the 2014/2024 rules differ.

### Spell Issues and Missing Effects

Spells:
- [ ] Shadow Blade just doesn't work

2014 Missing Effects
- [ ] Ensnaring Strike needs effect and damage activity
- [ ] Geas
- [ ] Phantasmal Killer

### Feature Issues and Missing Effects

- [ ] Wild Companion Spell uses spell slots and not wildshape uses (this is how its set up on DDB)
- [ ] Dancing Bards Agile Strikes not adjusted for agile property.
- [ ] Psionic Sorcery does not create spells or have a feature to consume spells

### Monster Feature Corrections Required

- [x] Clay Golem Haste is parsed as a bonus action.
- [x] Dullahan Battleaxe damage is not parsed correctly
- [x] Dullahan parses some legendary actions as mythic actions incorrectly
- [x] Duergar Warlord Psychic-Attuned Hammer damage is not correctly parsed
- [ ] Autumn Eladrin Cure Wounds is not cast as the correct level spell
- [x] Flumph Tendrils do not parse damage correctly
- [x] Living Blade of Disaster parses quite badly
- [ ] Nilbog, Reversal of Fortune parsing is not quite right
- [x] Nosferatu Bite does not parse correctly
- [x] A Dragon's Frightful presence does not generate a range and area shape
- [x] Monsters with Magic Weapons  feature do not always get the magic property applied to their weapons.

**Comment by carolinex** [comment]
Thank you so much for all the work! 
On my test munching I found an issue with these two items - **Needler Pistol** and **Paralysis Pistol** - which error out with message "Failed to find loot type for <name>". Both are from _Quests from the Infinite Staircase_.

```javascript
// Needler Pistol - data._id: "3W5DAaabtgQKuCoO"
{
    "id": 387,
    "baseTypeId": 2103445194,
    "definitionKey": "2103445194:387"
}

// Paralysis Pistol  - data._id: "0dTXYB8HBeRbwSo8"
{
    "id": 388,
    "baseTypeId": 2103445194,
    "definitionKey": "2103445194:388"
}
```
I noticed that on DnD Beyond the **Needler Pistol** has two links, one of them is returning a 404 error
- https://www.dndbeyond.com/equipment/366-needler-pistol
- https://www.dndbeyond.com/equipment/66-needler-pistol

**Paralysis Pistol** has only one "equipment" link
- https://www.dndbeyond.com/equipment/367-paralysis-pistol

Let me know if it helps you to gather any other parts of the json, I can definitely do that if needed.

**Comment by MrPrimate** [comment]
> Thank you so much for all the work! On my test munching I found an issue with these two items - **Needler Pistol** and **Paralysis Pistol** - which error out with message "Failed to find loot type for ". Both are from _Quests from the Infinite Staircase_.
> 
> ```js
> // Needler Pistol - data._id: "3W5DAaabtgQKuCoO"
> {
>     "id": 387,
>     "baseTypeId": 2103445194,
>     "definitionKey": "2103445194:387"
> }
> 
> // Paralysis Pistol  - data._id: "0dTXYB8HBeRbwSo8"
> {
>     "id": 388,
>     "baseTypeId": 2103445194,
>     "definitionKey": "2103445194:388"
> }
> ```
> 
> I noticed that on DnD Beyond the **Needler Pistol** has two links, one of them is returning a 404 error
> 
>     * https://www.dndbeyond.com/equipment/366-needler-pistol
> 
>     * https://www.dndbeyond.com/equipment/66-needler-pistol
> 
> 
> **Paralysis Pistol** has only one "equipment" link
> 
>     * https://www.dndbeyond.com/equipment/367-paralysis-pistol
> 
> 
> Let me know if it helps you to gather any other parts of the json, I can definitely do that if needed.

It should've imported these for you despite those error messages. Is that not the case?

**Comment by kraytGG** [comment]
I am receiving this issue when attempting to import a player.
![Capture](https://github.com/user-attachments/assets/5c73d92e-1a50-405d-ba3f-649f1e5bb574)

**Comment by lholden** [comment]
![image](https://github.com/user-attachments/assets/9aa60d8d-5e30-42c7-926d-0f3995ccca6a)
A minor issue. But the "D&D Beyond" icon on item sheets overlaps the close and configure buttons.

**Comment by MrPrimate** [comment]
> I am receiving this issue when attempting to import a player. ![Capture](https://private-user-images.githubusercontent.com/182274915/369531880-5c73d92e-1a50-405d-ba3f-649f1e5bb574.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4NjA2MDUsIm5iZiI6MTcyNjg2MDMwNSwicGF0aCI6Ii8xODIyNzQ5MTUvMzY5NTMxODgwLTVjNzNkOTJlLTFhNTAtNDA1ZC1iYTNmLTY0OWYxZTViYjU3NC5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyMFQxOTI1MDVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MzJiYzRhOGJlN2FmMGQzZTEyMjlmZTJmNGQxNzM5M2MzZWU2MzUxMGE4ODI3MzgzOWM5ZTc0MWNiOThmMjk5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.JqKrsBmCiGVrURzFddUvXKPhegVsVqFjqfuSTbKL8Gc)

This is fixed in 6.0.2 out in an hour or so.

**Comment by MrPrimate** [comment]
> ![image](https://private-user-images.githubusercontent.com/20953/369535367-9aa60d8d-5e30-42c7-926d-0f3995ccca6a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4NjE1NzYsIm5iZiI6MTcyNjg2MTI3NiwicGF0aCI6Ii8yMDk1My8zNjk1MzUzNjctOWFhNjBkOGQtNWUzMC00MmM3LTkyNmQtMGYzOTk1Y2NjYTZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIwVDE5NDExNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwODAxNzQxZDBkNzE3NzE4YzhlMGEyMTA0NDFiYjg5ZjlhNTJhZmYxNWMwYjQ3OWNhNDcwMmIxY2RmZjc4NDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WFAflVllODwvC_PCrKVgJuM95Cn-s6BjaJDw2g6yamc) A minor issue. But the "D&D Beyond" icon on item sheets overlaps the close and configure buttons.

Alas, this is beyond my CSS Skills. If anyone is reading this and knows how to fix it, PRs are welcome.

**Comment by lholden** [comment]
> Alas, this is beyond my CSS Skills. If anyone is reading this and knows how to fix it, PRs are welcome.

Could just use an image with a square aspect ratio? 

https://www.dndbeyond.com/content/1-0-2933-0/skins/waterdeep/images/dnd-beyond-b-red.png

**Comment by carolinex** [comment]
> > Thank you so much for all the work! On my test munching I found an issue with these two items - **Needler Pistol** and **Paralysis Pistol** - which error out with message "Failed to find loot type for ". Both are from _Quests from the Infinite Staircase_.
> > ```js
> > // Needler Pistol - data._id: "3W5DAaabtgQKuCoO"
> > {
> >     "id": 387,
> >     "baseTypeId": 2103445194,
> >     "definitionKey": "2103445194:387"
> > }
> > 
> > // Paralysis Pistol  - data._id: "0dTXYB8HBeRbwSo8"
> > {
> >     "id": 388,
> >     "baseTypeId": 2103445194,
> >     "definitionKey": "2103445194:388"
> > }
> > ```
> >   
> > I noticed that on DnD Beyond the **Needler Pistol** has two links, one of them is returning a 404 error
> > ```
> > * https://www.dndbeyond.com/equipment/366-needler-pistol
> > 
> > * https://www.dndbeyond.com/equipment/66-needler-pistol
> > ```
> > **Paralysis Pistol** has only one "equipment" link
> > ```
> > * https://www.dndbeyond.com/equipment/367-paralysis-pistol
> > ```

> > Let me know if it helps you to gather any other parts of the json, I can definitely do that if needed.
> It should've imported these for you despite those error messages. Is that not the case?

Yes, but it was added as a consumable trinket.

**Comment by carolinex** [comment]
> > ![image](https://private-user-images.githubusercontent.com/20953/369535367-9aa60d8d-5e30-42c7-926d-0f3995ccca6a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4NjE1NzYsIm5iZiI6MTcyNjg2MTI3NiwicGF0aCI6Ii8yMDk1My8zNjk1MzUzNjctOWFhNjBkOGQtNWUzMC00MmM3LTkyNmQtMGYzOTk1Y2NjYTZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIwVDE5NDExNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwODAxNzQxZDBkNzE3NzE4YzhlMGEyMTA0NDFiYjg5ZjlhNTJhZmYxNWMwYjQ3OWNhNDcwMmIxY2RmZjc4NDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WFAflVllODwvC_PCrKVgJuM95Cn-s6BjaJDw2g6yamc) A minor issue. But the "D&D Beyond" icon on item sheets overlaps the close and configure buttons.
> 
> Alas, this is beyond my CSS Skills. If anyone is reading this and knows how to fix it, PRs are welcome.

This works for me:
```css
a.header-button.control.ddb-open-url {
    width: auto;
    aspect-ratio: auto;
}
```

**Comment by MrPrimate** [comment]
> > > ![image](https://private-user-images.githubusercontent.com/20953/369535367-9aa60d8d-5e30-42c7-926d-0f3995ccca6a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjY4NjE1NzYsIm5iZiI6MTcyNjg2MTI3NiwicGF0aCI6Ii8yMDk1My8zNjk1MzUzNjctOWFhNjBkOGQtNWUzMC00MmM3LTkyNmQtMGYzOTk1Y2NjYTZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTIwVDE5NDExNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwODAxNzQxZDBkNzE3NzE4YzhlMGEyMTA0NDFiYjg5ZjlhNTJhZmYxNWMwYjQ3OWNhNDcwMmIxY2RmZjc4NDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WFAflVllODwvC_PCrKVgJuM95Cn-s6BjaJDw2g6yamc) A minor issue. But the "D&D Beyond" icon on item sheets overlaps the close and configure buttons.
> > 
> > Alas, this is beyond my CSS Skills. If anyone is reading this and knows how to fix it, PRs are welcome.
> 
> This works for me:
> ```css
> a.header-button.control.ddb-open-url {
>     width: auto;
>     aspect-ratio: auto;
> }
> ```
> 

Thank you!!!!

**Comment by crlgn** [comment]
I notice all spells have duplicated activities, and all emanation area spells are missing the template type. Are these known issues? Examples: Word of Radiance, Aura of Vitality. 
Let me know if there's anything you can outsource for these situations, I've been manually adding templates for a bunch of those spells and I can code so maybe I can help.

**Comment by MrPrimate** [comment]
> I notice all spells have duplicated activities, and all emanation area spells are missing the template type. Are these known issues? Examples: Word of Radiance, Aura of Vitality. Let me know if there's anything you can outsource for these situations, I've been manually adding templates for a bunch of those spells and I can code so maybe I can help.

The template is a known issue due to a data change at DDB and will be fixed in the next release.

The duplicate issue however I can not reproduce. did you much with the latest version and ensure you had update existing things checked?

**Comment by crlgn** [comment]
> > I notice all spells have duplicated activities, and all emanation area spells are missing the template type. Are these known issues? Examples: Word of Radiance, Aura of Vitality. Let me know if there's anything you can outsource for these situations, I've been manually adding templates for a bunch of those spells and I can code so maybe I can help.
> 
> The template is a known issue due to a data change at DDB and will be fixed in the next release.
> 
> The duplicate issue however I can not reproduce. did you much with the latest version and ensure you had update existing things checked?

Yes, I munched with that option on. My first unch was right on the first release of v6.0.0, maybe I should remove all spells and munch again?
<img width="300" alt="image" src="https://github.com/user-attachments/assets/70dee296-a841-4e75-883b-9318689f9a94">
<img width="300" alt="image" src="https://github.com/user-attachments/assets/ebc47821-acc0-499f-bc0d-f38f459217c8">
<img width="300" alt="image" src="https://github.com/user-attachments/assets/5bd38f8b-6a0b-4099-a9b6-b227b8cfda5d">

**Comment by oestred** [comment]
Bug - when importing a 2024 character, some inventory items look normal, but there are always several that take up many lines and look like this. 
![image](https://github.com/user-attachments/assets/8bb648bc-751b-4fa4-a4f2-958ed392b715)

**Comment by MrPrimate** [comment]
> Bug - when importing a 2024 character, some inventory items look normal, but there are always several that take up many lines and look like this. ![image](https://private-user-images.githubusercontent.com/74035873/375989756-8bb648bc-751b-4fa4-a4f2-958ed392b715.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4NTE4NjMsIm5iZiI6MTcyODg1MTU2MywicGF0aCI6Ii83NDAzNTg3My8zNzU5ODk3NTYtOGJiNjQ4YmMtNzUxYi00ZmE0LWE0ZjItOTU4ZWQzOTJiNzE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEzVDIwMzI0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4ZTkzNDcwNzJkODM1MTY4OTIwODFiMzFlMGI1ZjZlNmUwYTQwMzNiMTU0NWRjMjRjMmYxNGVlMWEwMTJiYjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WUFXOJJ08Xs37PjgDWA8sVOGm0ePM95SBetv5oyMpIY)

Can you:
- provide a public character url
- which character sheet are you using (default, legacy, or other)?
- does it do this with all other modules disabled?

**Comment by oestred** [comment]
> > Bug - when importing a 2024 character, some inventory items look normal, but there are always several that take up many lines and look like this. ![image](https://private-user-images.githubusercontent.com/74035873/375989756-8bb648bc-751b-4fa4-a4f2-958ed392b715.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg4NTE4NjMsIm5iZiI6MTcyODg1MTU2MywicGF0aCI6Ii83NDAzNTg3My8zNzU5ODk3NTYtOGJiNjQ4YmMtNzUxYi00ZmE0LWE0ZjItOTU4ZWQzOTJiNzE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDEzVDIwMzI0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4ZTkzNDcwNzJkODM1MTY4OTIwODFiMzFlMGI1ZjZlNmUwYTQwMzNiMTU0NWRjMjRjMmYxNGVlMWEwMTJiYjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.WUFXOJJ08Xs37PjgDWA8sVOGm0ePM95SBetv5oyMpIY)
> 
> Can you:
> 
> * provide a public character url
> * which character sheet are you using (default, legacy, or other)?
> * does it do this with all other modules disabled?

 Ah, good questions. I tried it without any modules and it worked on the default sheet.  I am using Tidy 5E sheet usually. It looks like Tidy 5E is doing that only with items in a container (backpack in this case) so it looks like a Tidy 5E bug.  I will report it there.

Thank you

**Comment by kgar** [comment]
In Tidy 5e Sheets, it looks like the window header / title bar option no longer works in dnd5e 4.0.x. I remember it working for Tidy sheets some time ago, but I haven't checked in a while. Here's what I'm getting now:

![image](https://github.com/user-attachments/assets/db983c24-16f2-4898-8568-8689f2608243)

![image](https://github.com/user-attachments/assets/84a07d16-c17d-47b1-afbc-1bb2df92103a)

If there's something I can do to assist, let me know.

Tidy PCs and NPCs are still using App V1 + Svelte from before.
Depending on dnd5e 4.1.x+ and Foundry V13+, Tidy may have to move all sheets to App V2, which could affect the class name, but the API functions for checking if a sheet is a Tidy character / NPC should always work, generally.

There is a sheet overhaul underway that will also use App V2 and provide new layouts for PCs, NPCs, vehicles, groups, and items, but it will be some time before it is ready.

**Comment by darkviruzz** [comment]
Hi, I'm not sure if this is the correct place to report this, if not, please tell me where i should report it instead

i noticed when importing a warlock, that the pact weapon creation is slightly bugged. because the active effect that is created uses 

system.proficient | Add | true
instead it should use
system.proficient | Override | 1

to generate a proficient pact weapon.

after manually changing it to override 1 it correctly works but with an update in beyond and reimport this change is gone again :(

**Comment by MrPrimate** [comment]
> Override

Should be fixed in the next release, thanks for the report.

**Comment by StefanEggenschwiler** [comment]
Hey

The item import worked fine with other books like the core rule books, Descent into Avernus, Tales from the Yawning Portal, etc. But for some reason it does not work for Tasha's Cauldron of Everything. None of the [items](https://www.dndbeyond.com/sources/dnd/tcoe/magic-items) seem to be imported, even when I specifically choose Tasha's as source filter. I use the latest versions of ddb-importer, DND and Foundry (v12).

![Image](https://github.com/user-attachments/assets/9262422e-9eaa-4965-8517-c654882166d1)

**Comment by MrPrimate** [comment]
> Hey
> 
> The item import worked fine with other books like the core rule books, Descent into Avernus, Tales from the Yawning Portal, etc. But for some reason it does not work for Tasha's Cauldron of Everything. None of the [items](https://www.dndbeyond.com/sources/dnd/tcoe/magic-items) seem to be imported, even when I specifically choose Tasha's as source filter. I use the latest versions of ddb-importer, DND and Foundry (v12).
> 
> Screenshot: Looking for [Nature's Mantle](https://www.dndbeyond.com/magic-items/2405507-natures-mantle) after running the item import.
> ![Image](https://github.com/user-attachments/assets/9262422e-9eaa-4965-8517-c654882166d1)

This is not a 2024 issue, please see https://docs.ddb.mrprimate.co.uk/docs/faqs/ddb-importer#spellsitems-that-i-have-on-ddb-dont-import and if you can't get them to import, please join out discord and we can troubleshoot from there

**Comment by Giannotts** [comment]
Hello, i have an issue that i can't resolve when i try to import a cleric PG above 4° Level i received this error.
hanks for help best regards

![Image](https://github.com/user-attachments/assets/b659b29d-8389-4d66-a149-ee3ea47f2874)

**Comment by MrPrimate** [comment]
> Hello, i have an issue that i can't resolve when i try to import a cleric PG above 4° Level i received this error.
> hanks for help best regards
> 
> ![Image](https://github.com/user-attachments/assets/b659b29d-8389-4d66-a149-ee3ea47f2874)

Please create a separate issue and include the public pc URL and try to answer the rear of the questions in the bug report template, thanks
